projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -128,7 +130,7 @@ public function akismet_logo_replacement_styles() {
 	 * @return int Indicating the relative ordering of module1 and module2.
 	 */
 	public static function sort_requires_connection_last( $module1, $module2 ) {
-		if ( $module1['requires_connection'] == $module2['requires_connection'] ) {
+		if ( (bool) $module1['requires_connection'] === (bool) $module2['requires_connection'] ) {
```

Comment:
Out of curiosity, does this do anything different than the fuzzy equiv?

Reply:
Depends on what the values are set to. If they're already always booleans, or 0/1, or something like that, then no. It does if for example `$module1['requires_connection']` is 1 while `$module2['requires_connection']` is 2 or something like that.

The code just below seems to be expecting it to be boolean, so this seemed like the expected behavior to me. Also grepping through the code it looks like things that set it are setting it to booleans.